### PR TITLE
add translucency to outlineColor

### PR DESCRIPTION
**Pull request by LHolst**
fix for https://github.com/CesiumGS/cesium/issues/8553

PR adds translucency to outlineColor in BillboardCollectionVS

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thank you so much for the pull request @LHolst! I noticed this is your first pull request and I wanted to say welcome to the Cesium community!

The [Pull Request Guidelines](https://github.com/CesiumGS/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines) is a handy reference for making sure your PR gets accepted quickly, so make sure to skim that.

* :x: Missing `CONTRIBUTORS.md` entry.
    * Please add yourself to the [contributors](https://github.com/LHolst/cesium/blob/patch-2/CONTRIBUTORS.md) file!
* :x: Missing CLA.
    * Please send in a [Contributor License Agreement](https://github.com/CesiumGS/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla) (CLA) and comment back here to let us know to check this!
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/patch-2/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/master/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

**Comment by OmarShehata** [comment]
Thanks @LHolst ! Can you please fill out a contributor's license agreement? https://github.com/CesiumGS/cesium/blob/master/CONTRIBUTING.md#contributor-license-agreement-cla

**Comment by LHolst** [comment]
CLA signed and CONTRIBUTORS updated

**Comment by hpinkos** [comment]
Thanks @LHolst!  I'll try to review this and get it in before the next release

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -154,6 +154,9 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for details on how to contribute to Cesiu
   - [Jonathan Nogueira](https://github.com/LuminousPath)
 - [Palantir Technologies, Inc.](https://palantir.com)
   - [Joey Rafidi](https://github.com/jrafidi)
+- [Geoplex GIS GmbH](https://www.geoplex.de)
+  - [Hannes Gräuler](https://github.com/lordi)
```
Comment (hpinkos) [inline CONTRIBUTORS.md:158]: It looks like this PR only has commits from `LHolst`.  Should this line be removed?

**Comment by hpinkos** [comment]
Looks great @LHolst, thanks!  I'll merge this as soon as CI passes

**Comment by LHolst** [comment]
Thanks @hpinkos !

**Pull request merged**